### ISSUE TITLE
Разметка каждого блока должна быть идентичной

### ISSUE THREAD

**Issue by frontgirl** [Sonrisa86/diplomaNetology#4]
https://github.com/Sonrisa86/diplomaNetology/blob/97f62d2fa5b1dedda2af0e16f695488344fe9a9c/index.html#L45

Названия классов должны быть одинаковы, за исключением цвета - за цвет должен отвечать дополнительный класс

**Comment by Sonrisa86** [comment]
не совсем поняла комментарий, вы хотите чтобы я задала вместо .athlets01, .athlets02 и .athlets03 одно название класса (к примеру .group-athlets) или чтобы добавила к ним это название класса?

**Comment by frontgirl** [comment]
Да, именно так. Например, .section , .section.section-violet и т.д. (названия написала для примера, можно и их взять, но можно подумать о более подходящих по смыслу). Причем для althletes и players разметка действительно одинакова - отличаются только надписи, картинки, и сторона (слева или справа большой отступ и картинка), а это можно задать  либо :nth-child(2n) как в блоке ниже, либо дополнительным классом на блок-секцию.

**Comment by Sonrisa86** [comment]
добавила .group-athlets и .group-players к блокам, и css поменяла соответственно. спасибо!
